File: csharp/WriteExample.cs
using System;
using System.Threading.Tasks;
using InfluxDB.Client;
using InfluxDB.Client.Api.Domain;
using InfluxDB.Client.Writes;

namespace Examples
{
    public class WriteExample
    {
        public static async Task Main(string[] args)
        {
            var url = "https://us-west-2-1.aws.cloud2.influxdata.com";
            var token = "my-token";
            var org = "my-org";

            using var client = InfluxDBClientFactory.Create(url, token);
            var writeApi = client.GetWriteApiAsync();

            var point = PointData.Measurement("weatherstation")
                .Tag("location", "San Francisco")
                .Field("temperature", 25.5)
                .Timestamp(DateTime.UtcNow, WritePrecision.Ns);

             await writeApi.WritePointAsync(point, "my-bucket", org);
        }
    }
}

File: python/WriteExample.py
from datetime import datetime

from influxdb_client import WritePrecision, InfluxDBClient, Point
from influxdb_client.client.write_api import SYNCHRONOUS

url = 'https://us-west-2-1.aws.cloud2.influxdata.com'
token = 'my-token'
org = 'my-org'
bucket = 'my-bucket'

with InfluxDBClient(url=url, token=token, org=org) as client:
    p = Point("weatherstation") \
        .tag("location", "San Francisco") \
        .field("temperature", 25.9) \
        .time(datetime.utcnow(), WritePrecision.MS)

    with client.write_api(write_options=SYNCHRONOUS) as write_api:
        write_api.write(bucket=bucket, record=p)
